src/main/java/nextstep/auth/authentication/session/SessionAuthenticationInterceptor.java

```diff
@@ -0,0 +1,27 @@
+package nextstep.auth.authentication.session;
+
+import nextstep.auth.authentication.AuthenticationInterceptor;
+import nextstep.auth.authentication.UserDetailService;
+import nextstep.auth.context.Authentication;
+import nextstep.auth.context.SecurityContext;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import java.io.IOException;
+
+import static nextstep.auth.context.SecurityContextHolder.SPRING_SECURITY_CONTEXT_KEY;
+
+public class SessionAuthenticationInterceptor extends AuthenticationInterceptor {
+
+    public SessionAuthenticationInterceptor(UserDetailService customUserDetailsService, SessionAuthenticationConverter authenticationConverter) {
```

Comment:
매개변수 타입을 좀 더 추상화 해서 받을 수 있을 것 같아요. 
```suggestion
    public SessionAuthenticationInterceptor(UserDetailService customUserDetailsService, AuthenticationConverter authenticationConverter) {
```
항상 구체화보다는 추상화에 의존하도록 하는게 유연성확보에 좋습니다 😄